charts/ceph-csi-rbd/README.md

```diff
@@ -112,6 +112,7 @@ charts and their default values.
 | `provisioner.skipForceFlatten`                 | Skip image flattening if kernel support mapping of rbd images which has the deep-flatten feature                                                     | `false`                                            |
 | `provisioner.timeout`                          | GRPC timeout for waiting for creation or deletion of a volume                                                                                        | `60s`                                              |
 | `provisioner.clustername`                      | Cluster name to set on the RBD image                                                                                                                 | ""                                                 |
+| `provisioner.setmetadata`                      | Set metadata on volume                                                                                                                               | `false`                                            |
```

Comment:
default is true in helm charts

Reply:
@Madhu-1 this is addressed now, thanks!

Reply:
it should contain internal default that is false ? 

	flag.BoolVar(&conf.SetMetadata, "setmetadata", false, "set metadata on the volume")

Reply:
Hmm, right!

Reply:
Sorry got confused, I think its values from charts!

Reply:
AFAIK this is the documentation of the default values in values.yaml.

Reply:
I believe it should internal default value since this will be used when no value in set for a variable in values.yaml.

Reply:
Lets come to a conclusion, we need this PR merged ASAP.